docs/overview_and_guide.rst

```diff
@@ -172,14 +172,16 @@ After account creation, launch Raiden with the path of your keystore supplied::
 Using --eth-rpc-endpoint/Infura
 *******************************
 
+.. warning::
```

Comment:
Shouldn't we also add this to the [requirements for safe usage](https://raiden-network.readthedocs.io/en/latest/overview_and_guide.html#requirements-for-safe-usage)?

Reply:
I'm not sure, we should fix that list before expanding it.